libraries/AP_HAL_ChibiOS/RCOutput.cpp

```diff
@@ -596,6 +598,29 @@ bool RCOutput::mode_requires_dma(enum output_mode mode) const
 #endif //#ifdef DISABLE_DSHOT
 }
 
+void RCOutput::print_group_setup_error(pwm_group &group, const char* error_string)
+{
+#ifndef HAL_NO_UARTDRIVER
+    uint8_t min_chan = 255;
+    uint8_t max_chan = 0;
+    for (uint8_t j = 0; j < 4; j++) {
+        uint8_t chan = group.chan[j];
+        if (chan >= max_channels) {
```

Comment:
I think generally in this code the pattern is to check against CHAN_DISABLED

Reply:
fixed, thanks.